src/resources/parser/glb-parser.js

```diff
@@ -0,0 +1,1466 @@
+            //             if (dracoAttribs.hasOwnProperty('TANGENT'))
+            //                 tangents  = extractAttribute(dracoAttribs.TANGENT);
+            //             if (dracoAttribs.hasOwnProperty('TEXCOORD_0'))
+            //                 texCoord0 = extractAttribute(dracoAttribs.TEXCOORD_0);
+            //             if (dracoAttribs.hasOwnProperty('TEXCOORD_1'))
+            //                 texCoord1 = extractAttribute(dracoAttribs.TEXCOORD_1);
+            //             if (dracoAttribs.hasOwnProperty('COLOR_0'))
+            //                 colors    = extractAttribute(dracoAttribs.COLOR_0);
+            //             if (dracoAttribs.hasOwnProperty('JOINTS_0'))
+            //                 joints    = extractAttribute(dracoAttribs.JOINTS_0);
+            //             if (dracoAttribs.hasOwnProperty('WEIGHTS_0'))
+            //                 weights   = extractAttribute(dracoAttribs.WEIGHTS_0);
+            //         }
+            //
+            //         if (geometryType == decoderModule.TRIANGULAR_MESH) {
+            //             var face = new decoderModule.DracoInt32Array();
+            //             indices = (numPoints > 65535) ? new Uint32Array(numFaces * 3) : new Uint16Array(numFaces * 3);
+            //             for (i = 0; i < numFaces; ++i) {
+            //                 decoder.GetFaceFromMesh(outputGeometry, i, face);
+            //                 indices[i * 3]     = face.GetValue(0);
+            //                 indices[i * 3 + 1] = face.GetValue(1);
+            //                 indices[i * 3 + 2] = face.GetValue(2);
+            //             }
+            //             decoderModule.destroy(face);
+            //         }
+            //
+            //         decoderModule.destroy(outputGeometry);
+            //         decoderModule.destroy(decoder);
+            //         decoderModule.destroy(buffer);
+            //     }
+            // }
+            // //
+
+            // get indices
+            var indices =
+                primitive.hasOwnProperty('indices') ?
+                    getAccessorData(accessors[primitive.indices], bufferViews, buffers) : null;
+
+            var vertexBuffer = createVertexBuffer(device, attributes, indices, accessors, bufferViews, buffers);
+
+            var mesh = new pc.Mesh();
+            mesh.vertexBuffer = vertexBuffer;
+            mesh.primitive[0].type = getPrimitiveType(primitive);
+            mesh.primitive[0].base = 0;
+            mesh.primitive[0].indexed = (indices !== null);
+            if (indices !== null) {
+                var indexFormat;
+                if (indices instanceof Uint8Array) {
+                    indexFormat = pc.INDEXFORMAT_UINT8;
+                } else if (indices instanceof Uint16Array) {
+                    indexFormat = pc.INDEXFORMAT_UINT16;
+                } else {
+                    indexFormat = pc.INDEXFORMAT_UINT32;
```

Comment:
Minor point, but 32-bit indices are not guaranteed to be supported on a very small percentage of WebGL 1.0 devices (Android).

Reply:
will add a TODO here.